snap/snapcraft.yaml

```diff
@@ -1,55 +1,58 @@
-name: srsran
+name: srslte
```

Comment:
Why the name change?

Reply:
it's the name that was used in `release_20_10` so some directories created within the snap would have the name `srslte` rather than `srsran` like `/usr/share/srslte` where the config files are stored.
So to have everything in sync I renamed it :)

Reply:
Ok, thanks!